client/src/globals.d.ts

```diff
@@ -0,0 +1,28 @@
+interface AdobeS {
+  // Configuration properties
+  trackExternalLinks: boolean;
+
+  // Variables to set when tracking
+  linkTrackVars: string;
+  linkTrackEvents: string;
+  events: string;
+  pageURL: string;
+  eVar26: string;
+  eVar27: string;
+
+  // Tracking functions
+  t: () => void;
+  tl: (
+    linkObject: true | undefined,
+    linkType: string,
+    linkName: string,
+  ) => void;
+}
+
+interface AWSCShortbreadObject {
+  checkForCookieConsent: () => void;
+}
+
+declare const s: AdobeS;
```

Comment:
*nitpick (non-blocking)*:

It's a difficult to know what `t`, `tl`, and `s` are supposed to mean as an outside contributor. Can we open a follow-up issue to replace these with more meaningful variable names?

Reply:
Heh, if only...  Don't worry, we've tried 😆

For real, this is just **required by Adobe Analytics**.  The meanings are all provided in their docs, but our internal doc can help provide a lot more detail ("Amplify Documentation: Adobe Analytics Dev Guide").

Reply:
Thanks! I just looked through the docs. It makes sense where these are coming from now!